validator/client/validator.go

```diff
@@ -603,6 +604,16 @@ func (v *validator) UpdateDuties(ctx context.Context, slot primitives.Slot) erro
 		return err
 	}
 
+	allExitedCounter := 0
+	for i := range resp.CurrentEpochDuties {
+		if resp.CurrentEpochDuties[i].Status == ethpb.ValidatorStatus_EXITED {
+			allExitedCounter++
+		}
+	}
+	if allExitedCounter != 0 && allExitedCounter == len(resp.CurrentEpochDuties) {
```

Comment:
why do we need `allExitedCounter != 0`? 

`allExitedCounter == len(resp.CurrentEpochDuties)` seems sufficient

Reply:
if they are both equal to 0 it'd pass, I guess it's just not displaying that log if there's no keys